src/app/admin/Smaregi/Index.vue

```diff
@@ -113,7 +113,7 @@
           <div class="mt-4">
             <button @click="isEdit = true">
               <div class="inline-flex h-12 items-center justify-center rounded-full bg-op-teal px-6 shadow min-w-32">
-                <span class="text-base font-bold text-white">編集</span>
+                <span class="text-base font-bold text-white">{{$t("editCommon.edit")}}</span>
```

Comment:
```suggestion
                <span class="text-base font-bold text-white">{{ $t("editCommon.edit") }}</span>
```

細かいですが、上記のように `{{`, `}}` の前後にスペースがあると、その他のコードとの統一性が取れるかな、と思いました。

そうなっていないコードもありますが・・・

Reply:
そのあたりはformatで対応ですね。